tests/unit/test/plan/grammar/test_data_types.py

```diff
@@ -46,6 +46,11 @@ def test_copy(self):
         self.assertEqual(copy.copy(data_types.AnyValue('str')), 'Hello')
         self.assertEqual(copy.deepcopy(data_types.AnyValue('str')), 'Hello')
 
+    def test_datetime(self):
+        self.assertEqual(data_types.AnyValue('datetime'), '2018-09-01T00:00:00Z')
+        self.assertNotEqual(data_types.AnyValue('datetime'), 'Hello')
+        self.assertNotEqual(data_types.AnyValue('datetime'), 15)
+
```

Comment:
datetime is accounted for, but shouldn't there be tests for `date` as well?

Reply:
You are right! Working on that!

Reply:
Added @jeff-eb . Thanks for your review